test/PowerShellEditorServices.Test/Console/InputPromptHandlerTests.cs

```diff
@@ -81,17 +83,18 @@ public void InputPromptHandlerAcceptsArrayOfNonStringValues()
                     CancellationToken.None);
 
             Assert.Equal("Numbers[0]", inputPromptHandler.LastField.Name);
-            inputPromptHandler.ReturnInputString("1");
-            inputPromptHandler.ReturnInputString("");
+            await inputPromptHandler.ReturnInputString("1");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)